src/main/java/fruitshop/Main.java

```diff
@@ -0,0 +1,50 @@
+package fruitshop;
+
+import fruitshop.dao.FruitStorageDaoImpl;
+import fruitshop.model.dto.FruitOperationDto;
+import fruitshop.model.dto.ReportDto;
+import fruitshop.service.FruitDtoParser;
+import fruitshop.service.FruitDtoParserImpl;
+import fruitshop.service.FruitService;
+import fruitshop.service.FruitServiceImpl;
+import fruitshop.service.ReportService;
+import fruitshop.service.ReportServiceImpl;
+import fruitshop.service.file.ReadService;
+import fruitshop.service.file.ReadServiceImpl;
+import fruitshop.service.file.WriteService;
+import fruitshop.service.file.WriteServiceImpl;
+import fruitshop.service.shopoperation.OperationHandler;
+import fruitshop.service.shopoperation.OperationType;
+import fruitshop.service.shopoperation.StorageDecreaseHandler;
+import fruitshop.service.shopoperation.StorageIncreaseHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Map<OperationType, OperationHandler> operationMap = new HashMap<>();
+        operationMap.put(OperationType.BALANCE,
+                new StorageIncreaseHandler(new FruitStorageDaoImpl()));
```

Comment:
declare FruitStorageDaoImpl as a seperate local variable, we don't need 4 different instances

Reply:
done